packages/core/src/amazonqGumby/chat/controller/controller.ts

```diff
@@ -303,10 +302,10 @@ export class GumbyController {
         await this.prepareProjectForSubmission(message)
     }
 
-    private async transformationFinished(message: { tabID: string; jobStatus: string }) {
+    private async transformationFinished(tabID: string, jobStatus: string = '') {
```

Comment:
See my other comment about error messages. Going to eventually use the jobStatus to customize the error message shown to user. For now, include it here with a default value of empty string since it is not used.